src/preview.js

```diff
@@ -1,4 +1,5 @@
 import Cookies from './cookies';
+import { config } from './index';
```

Comment:
Instead of having circular dependency (index.js -> toolbar.js -> preview.js -> index.js)

Reply:
I would prefer you add dependency to `Share` call `Share.close` (that actually runs `config.corsLink.contentWindow.postMessage({...`)